lib/sous-chef.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * This class is designed to parse the "What's cooking" mails sent to
+ * the Git mailing list about twice a week by the Git maintainer.
+ */
+
+export class SousChef {
+    public readonly mbox: string;
+    public readonly messageID: string | undefined;
+    public readonly branches: {
+        [branchName: string]: {
```

Comment:
Another chance to use `Map`?  One reason is that iteration looks simpler in the code.

Reply:
I am using a `Map` now, but I do not see any iteration that would be simplified ;-)

Reply:
> I am using a `Map` now, but I do not see any iteration that would be simplified ;-)

I thought maybe you could use a different `for` loop:
```
for (const [branchName, branch] of sousChef.branches)
... branch.text.replace ...
```